### ISSUE TITLE
crash on pyBigWig.open

### ISSUE THREAD

**Issue by balwierz** [vaquerizaslab/fanc#2]
I was running in auto mode. 
There are no bigWig files in the output directory, only sam and sorted sam files.
The temporary directory does not exist anymore.

```
2020-02-09 05:26:59,352 INFO Cannot find sambamba on this machine, falling back to samtools sort. This is not a problem, but if you want to speed up your SAM/BAM file sorting, install [...]
2020-02-09 06:59:11,503 INFO Getting genome regions (fragments or bins)
2020-02-09 06:59:11,503 INFO Getting regions
[bwHdrRead] There was an error while reading in the header!
[pyBwOpen] bw is NULL!
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/dist-packages/genomic_regions/regions.py", line 183, in load
    f = pyBigWig.open(file_name, 'r')
RuntimeError: Received an error during file opening!
```

**Comment by kaukrise** [comment]
Hi, sorry, the error you are seeing is misleading. The load function tries different file types until it can find one that can be opened. Unfortunately the pyBwOpen error messages originate from pyBigWig and cannot be suppressed (at least I did not find a way).
This seems to be an issues with loading your genome file. What format of the genome file are you supplying? Did you run `fanc` on the examples here on GitHub and did you see the same issue?

**Comment by balwierz** [comment]
Thanks
This might be related to #1 although #1 appeared after #2 this error.
I am ungzipping genomic fasta then and trying again.

**Comment by kaukrise** [comment]
Cool, let me know if the issue persists. I'll see if I can make that error message more explicit for the next version

**Comment by balwierz** [comment]
ungzipping helped

**Comment by kaukrise** [comment]
Fixed in 0.8.18. Also update to genomic_regions 0.0.6, which has a more graceful way of checking for BigWig files.
